FirebasePerformance/Sources/AppActivity/FPRAppActivityTracker.m

```diff
@@ -24,7 +24,11 @@
 #import "FirebasePerformance/Sources/Timer/FIRTrace+Internal.h"
 #import "FirebasePerformance/Sources/Timer/FIRTrace+Private.h"
 
+#import <GoogleUtilities/GULAppEnvironmentUtil.h>
+
 static NSDate *appStartTime = nil;
+static NSDate *doubleDispatchTime = nil;
+static NSDate *applicationFinishLaunchTime = nil;
```

Comment:
can we change this to applicationDidFinishLaunchTime? (To avoid the ambiguity of applicationDidFinishLaunchTime and applicationWillFinishLaunchTime)